### DISCUSSION TITLE
posts/007_legends_developer_eurocom_lays_75_percent_workforce/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#25] [General]
# posts/007_legends_developer_eurocom_lays_75_percent_workforce/

Evaluation Only. Created with Aspose.Words. Copyright 2003-2022 Aspose Pty Ltd.
джерелоhttps://www.gamesradar.com/007-legends-developer-eurocom-lays-75-percent-workforce/ Title:007 Розробник Legends Eurocom звільняє 75 відсотків робочої сили | GamesRadar+
007 Legends, розробник Eurocom звільняє 75 відсотків робочої силиРозробник 007 Legends і давній ліцензований ігровий підрядник Eurocom звільнив щонайменше 150 співробітників у рамках широкомасштабної реструктуризації. GamesIndustry International повідомляє, що студія, розташована в Дербі, Англія, продовжить працювати лише з 50 співробітниками, порівняно з понад 200 співробітниками.
«Eurocom проводить реструктуризацію, яка, на жаль, означала, що сьогодні ми звільнили більшість нашої робочої сили», — заявив минулого тижня директор студії Г’ю Бінс.

https://spielradar.com/posts/007_legends_developer_eurocom_lays_75_percent_workforce/

**Comment by ukrskiff** [comment]
test
